assets/js/editor-components/sidebar-compatibility-notice/style.scss

```diff
@@ -0,0 +1,13 @@
+.wc-blocks-sidebar-compatibility-notice {
+	border-top: 1px solid $gray-200;
```

Comment:
There is no `border-top` from my side!

<img width="297" alt="image" src="https://user-images.githubusercontent.com/14235870/183071172-ccbef80e-a192-483e-8948-bd4486435278.png">